lib/rubocop/cop/style/arguments_forwarding.rb

```diff
@@ -212,7 +212,8 @@ def register_forward_all_offense(def_or_send, send_or_arguments, rest_or_splat)
         end
 
         def arguments_range(node, first_node)
-          arguments = node.arguments
+          additional_arg_type = %i[lvar arg].freeze
```

Comment:
Can you make it a constant and make it a plural name?

Reply:
I updated this PR.